MadnessTests/ConcatenationTests.swift

```diff
@@ -42,11 +42,6 @@ func assertTree<C: CollectionType, T>(parser: Parser<C, T>.Function, input: C, m
 	return Assertions.assert(parsed?.0, match, tree, message: "should have parsed \(input) as \(tree). " + message, file: file, line: line)
 }
 
-func assertTree<C: CollectionType, T>(parser: Parser<C, T>.Function, input: C, match: ((T, C.Index), T) -> Bool, tree: T, message: String = "", file: String = __FILE__, line: UInt = __LINE__) -> (T, C.Index)? {
-	let parsed: Parser<C, T>.Result = parser(input, input.startIndex)
-	return Assertions.assert(parsed, match, tree, message: "should have parsed \(input) as \(tree). " + message, file: file, line: line)
-}
-
```

Comment:
:+1: to removing this. I’d eventually like to :fire: the rest of these, so this is helpful!